node/overseer/src/lib.rs

```diff
@@ -746,12 +785,28 @@ where
 						Event::Stop => {
 							self.stop().await;
 							return Ok(());
-						}
+						},
+						Event::BlockImported(block) if self.sync_oracle.is_syncing() => {
```

Comment:
Why do we still need the oracle? If I understand correctly, we don't get any `BlockImported` events anyway during sync.